spring-grpc-docs/src/main/antora/modules/ROOT/partials/_configprops.adoc

```diff
@@ -36,4 +36,4 @@
 |spring.grpc.server.ssl.enabled |  | Whether to enable SSL support. Enabled automatically if "bundle" is provided unless specified otherwise.
 |spring.grpc.server.ssl.secure | `+++true+++` | Flag to indicate that client authentication is secure (i.e. certificates are checked). Do not set this to false in production.
 
-|===
+|===
```

Comment:
Sorry, **[UNRELATED]** from the formatter and it keeps doing it so I wanted to just check it so I can quite rolling it back before commits.